src/test/regress/isolation_schedule

```diff
@@ -2,3 +2,4 @@ test: isolation_cluster_management
 test: isolation_dml_vs_repair
 test: isolation_concurrent_dml
 test: isolation_drop_shards
+test: isolation_add_node_vs_copy
```

Comment:
We probably want to do this test before the `isolation_cluster_management` test and do a `SELECT master_remove_node(nodename, nodeport) FROM pg_dist_node` in the `teardown`